examples/js/app.ts

```diff
@@ -16,7 +15,10 @@ const searchClient = algoliasearch(
 
 type QuerySuggestionHit = { query: string };
 
-const recentSearches = createRecentSearchesPlugin({ key: 'recent' });
+const recentSearches = createSearchableLocalStorageRecentSearchesPlugin({
+  key: 'recent',
```

Comment:
we might want to use a less-likely to clash name in the example. Maybe Algolia-recent or something?

Reply:
Note that the [local storage item is prefixed by us](https://github.com/algolia/autocomplete.js/blob/89efd85ead8e3ee16a914af423c4a7e84e318da9/packages/autocomplete-plugin-recent-searches/src/createLocalStorageRecentSearchesPlugin.ts#L43) so this key must only be unique user-land.